python/ray/data/_internal/execution/operators/map_operator.py

```diff
@@ -427,6 +433,10 @@ def num_active_tasks(self) -> int:
         #   to reflect the actual data processing tasks.
         return len(self._data_tasks)
 
+    @property
+    def estimated_output_num_rows(self) -> Optional[int]:
+        return getattr(self, "_estimated_output_num_rows", 0)
+
```

Comment:
shouldn't need this once we implement class attribute at the `PhysicalOperator` level.